fe/fe-core/src/main/java/com/starrocks/backup/BackupHandler.java

```diff
@@ -416,7 +416,11 @@ private void restore(Repository repository, Database db, RestoreStmt stmt) throw
         // Also remove all unrelated objs
         Preconditions.checkState(infos.size() == 1);
         BackupJobInfo jobInfo = infos.get(0);
-        checkAndFilterRestoreObjsExistInSnapshot(jobInfo, stmt.getTableRefs());
+        // If TableRefs is empty, it means that we do not specify any table in Restore stmt.
```

Comment:
```
BACKUP SNAPSHOT test.testbackup TO test_repo PROPERTIES ("type" = "full"); RESTORE SNAPSHOT test.testbackup FROM test_repo PROPERTIES("backup_timestamp" = "xxx", "replication_num" = "1");
```
The example is incorrect in the commit message. `test.testbackup` is a table name, but you are backing up a database.

Reply:
There is some confusion here. Through the official document test.testbackup means that [db_name].{snapshot_name} which is the name represents a single snopshot in the repository but not the table name.
The full stmt for backup is like:

BACKUP SNAPSHOT [db_name].{snapshot_name}
TO `repository_name`
ON (
`table_name` [PARTITION (`p1`, ...)],
...
)
PROPERTIES ("key"="value", ...);

So, the tables' name are in the 'ON...' clause. And if we ignore it, we get the database-level backup. This the same for restore.